### PR TITLE
update plugin

### PR DESCRIPTION
**Pull request by azazellochg**
- add 0.2.3 support
-func def are now similar to cryolo plugin
- add denoise step

### CODE REVIEW

**Comment by pconesa** [comment]
@azazellochg , there was already a python3_migration banch....I think it is better to send changes there and later make it devel.

File: setup.py [hunk 1]
```diff
@@ -40,7 +40,7 @@
     # For a discussion on single-sourcing the version across setup.py and the
     # project code, see
     # https://packaging.python.org/en/latest/single_source_version.html
-    version='1.1',  # Required
+    version='1.2',  # Required
```
Comment (JorMaister) [inline setup.py]: As @pconesa mentioned, we're updating the version number of all plugins in scipion3 to make them begin in v3.0.0.

Comment (azazellochg) [reply setup.py]: sure, go ahead

**Comment by azazellochg** [comment]
> @azazellochg , there was already a python3_migration banch....I think it is better to send changes there and later make it devel.

I didnt get this.. This pr is to py3 branch

File: setup.py (hunk 1 above)
Comment (azazellochg) [reply setup.py]: fixed in 2404e6f

**Review by pconesa — APPROVED** [review]

**Pull request merged**